src/Componet/Header.jsx

```diff
@@ -1,24 +1,36 @@
 import React, { useState } from 'react'
-import { Link} from "react-router-dom";
-import {FaInstagram, FaLinkedin} from "react-icons/fa"
+import { Link } from "react-router-dom";
+import { FaInstagram, FaLinkedin } from "react-icons/fa"
 import "./header.css"
 import Register from '../Register';
 
 export default function Header() {
-    const[Open,setOpen]= useState(false);
+  const [Open, setOpen] = useState(false);
+  const [login, setLogin] = useState(false);
+
+  const logged = () => {
+    return "logado"
+  }
+  const logoff = () => {
+    return "Deslogado..."
+  }
 
 
   return (
     <div className='link'>
-        <Link style={{textDecoration:" none"}} to="/home">Home</Link>
-        <Link style={{textDecoration:" none"}} to="/contact">Contato</Link>
-        <button 
+      <div className='div-log'>{login ? logged() : logoff()}</div>
```

Comment:
como falei antes, tá sem propósito essas funções

```suggestion
      <div className='div-log'>{login ? "logado" :"Deslogado..."}</div>
```